### ISSUE TITLE
Inaccessible Warning Messages

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/hourglass#354]
## Report Issue
**Describe the inaccessible feature**
 for example the warning for an incorrect username and password combination, can be dismissed by the user, but does auto-clear after a short delay. There is no warning that it is about to disappear. 

The shortness of time before it disappeared made it impossible to inspect the element involved. 

But the message also causes the screen to scroll so that it is visible (as it appears at the top of the page). This is problematic in itself - the message is not near the input that triggers it, and the screen moves unexpectedly.

see #333 

**Which users are impacted?**

 1. Everyone

**To Reproduce**
Steps to reproduce:
1. Attempt to log in with an incorrect username:password
5. This is more obvious at higher magnifications where the top of the page is not visible on entering the password, as the warning message appears at the top, and therefore the page will suddenly scroll up to it.  At lower magnifications with the top of the page onscreen throughout, this is less obvious.

**Screenshots**

## Preferred Solution
- warning message should persist until dismissed by user
- warning message should appear at top of screen, not at top of page - i.e. not cause the page to scroll
- warning message link to what caused it should be programmatically determined - see https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA22.html

## Additional context
These are also causing a change in context, as per 3.2 Predictable. See (see https://github.com/BirkbeckCTP/hourglass/issues/341) for discussion, and the review should also consider whether they can be programmatically determined as per 4.1 Compatible (see https://github.com/BirkbeckCTP/hourglass/issues/331).
